include/alpaka/mem/view/Traits.hpp

```diff
@@ -240,9 +240,8 @@ namespace alpaka
         static_assert(
             Dim<TView>::value == Dim<TExtent>::value,
             "The view and the extent are required to have the same dimensionality!");
-        static_assert(
-            meta::IsIntegralSuperset<Idx<TView>, Idx<TExtent>>::value,
-            "The view and the extent must have compatible index types!");
+
+        assert((extent <= getExtents(view)).all() && "The memset extent must not be larger than the view extents!");
```

Comment:
```suggestion
        assert((extent <= getExtents(view)).all() && "The memset extent must not be larger than the view extent!");
```
and also for the others below

Reply:
done